spec/ParseObject.spec.js

```diff
@@ -268,16 +268,28 @@ describe('Parse.Object testing', () => {
     });
   });
 
-  it_exclude_dbs(['postgres'])("can set null", function(done) {
+  it("can set null", function(done) {
```

Comment:
Probably you should be able to set null on all things in PG. We had this discussion about validations and how non-nullability should be a validation, but I think for now, its best to aim for API incompatibilities only when truly unavoidable